pkg/builder/build.go

```diff
@@ -230,3 +234,22 @@ func (blder *Builder) doController(r reconcile.Reconciler) error {
 	blder.ctrl, err = newController(name, blder.mgr, controller.Options{Reconciler: r})
 	return err
 }
+
+func (blder *Builder) doWebhook() error {
+	svr := &webhook.Server{}
+	err := svr.EnableWebhooksFor(blder.apiType)
+	// If no error, we add the Server instance to the controller manager.
+	// If we get a NotImplementDefaulterValidatorInterfacesError, we discord the Server instance.
+	// If other type of error, surface it.
+	if err == nil {
+		e := blder.mgr.Add(svr)
```

Comment:
quick question: shouldn't this be done before calling `svr.EnableWebhooksFor` anyways to ensure mgr injects dependencies for webhooks (mainly scheme) ?